### PR TITLE
Save form updates

### PR DESCRIPTION
**Pull request by Erin007**
This PR includes a few incremental updates to move the current state of the Save Panel closure to the proposed design: 

1.) Collapsible section for column descriptions, including column count and whether each column is a feature or label 
2.) Remove checkboxes for misuses and instead include those ideas in the misuses prompt 
3.) Adjust width of uses/misuses input boxes to encourage more detailed user input 

BEFORE: 

https://user-images.githubusercontent.com/12300669/111186095-0a9ce400-8589-11eb-9d37-023d11cda2d4.mov

AFTER: 

https://user-images.githubusercontent.com/12300669/111185677-9b26f480-8588-11eb-94f4-20eafab44a55.mov

DESIGN COMP: 

Screen Shot 2021-03-15 at 12 14 11 PM

### CODE REVIEW

**Comment by made-line** [comment]
Screen Shot 2021-03-15 at 12 52 18 PM

I like how it cleans up the screen! 

On my end, the caret doesn't show up. Not sure if it's something on my end, but I'll ask about in our meeting.

File: src/UIComponents/SaveModel.jsx [hunk 1]
```diff
@@ -15,24 +15,40 @@ class SaveModel extends Component {
...
-
     for (const columnDescription of this.props.columnDescriptions) {
+      const columnType = columnDescription.id === this.props.labelColumn
+        ? "label" : "feature";
       fields.push({
         id: columnDescription.id,
         isColumn: true,
-        text: "Description for: " + columnDescription.id,
-        answer: columnDescription.description
+        text: "Description for: " + columnDescription.id + ` (${columnType})`,
```
Comment (breville) [inline src/UIComponents/SaveModel.jsx:43]: It's a bit odd to mix `+` and string interpolation in one line.. maybe the whole thing could be generated using string interpolation.

File: src/UIComponents/SaveModel.jsx [hunk 2]
```diff
@@ -15,24 +15,40 @@ class SaveModel extends Component {
...
+        answer: columnDescription.description,
```
Comment (breville) [inline src/UIComponents/SaveModel.jsx:44]: stray comma?

File: src/constants.js [hunk 3]
```diff
@@ -363,5 +363,9 @@ export const styles = {
   floatLeft: {
     float: "left",
     margin: 20
+  },
+
+  ninetyFiveWidth: {
```
Comment (breville) [inline src/constants.js:368]: Can we name a bit more for its purpose and less for the exact number? :)

File: src/UIComponents/SaveModel.jsx [hunk 4]
```diff
@@ -41,64 +57,96 @@ class SaveModel extends Component {
...
-                  )}
+            <div key={nameField.id} style={styles.cardRow}>
+              <label>{nameField.text}</label>
+              <div>
+                <textarea
+                  rows="1"
+                  onChange={event =>
+                    this.handleChange(event, nameField.id, nameField.isColumn)
+                  }
+                />
```
Comment (breville) [inline src/UIComponents/SaveModel.jsx:91]: The name field still seems to be vertically expandable and supports multiple rows of text.  Can we get it to be a single line only?

**Review by breville — APPROVED** [review]

**Pull request merged**